scripts/leetcode.js

```diff
@@ -368,6 +368,91 @@ function findCode(
   }
 }
 
+const getCodeAndMsg = async (msg) => {
```

Comment:
you're creating a lot of tech debt with these two new functions. also, it doesn't incorporate updating local windows storage when you've already solved the problem before. Solving same problem keep increasing up your problem count tally.